tests/LocalBearTestHelper.py

```diff
@@ -14,6 +14,14 @@
 
 @contextmanager
 def execute_bear(bear, *args, **kwargs):
+    """
+    Execute the bear routine.
+
```

Comment:
sorry. I added an extra file in first push, fixed that and then an ``autopep8`` warning showed up after pushing again.